.changeset/slimy-mails-clean.md

```diff
@@ -1,5 +1,5 @@
 ---
-"@lit-labs/virtualizer": patch
+'@lit-labs/virtualizer': patch
```

Comment:
i wonder if edits to these markdown files might change the PR and commit that get linked with this update in the changelog. is it possible to exclude `.changeset/` from formatting?

Reply:
I hope that it would associate the PR that introduced the changeset but I'm not sure. I forgot to add one specifically for this PR so at a minimum these edits are tripping up the "did you add a changeset?" check.

Reply:
ah right. also new files would probably get formatted as they go in so likely not a big deal.

Reply:
Yeah, the workflow update should enforce that they're formatted properly from now on.